### ISSUE TITLE
Can't switch to Retribution Aura inside dungeons

### ISSUE THREAD

**Issue by cjwyatt** [dalaranwow/dalaran-wow#5095]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Go into a dungeon with retribution aura on and note it's in your list of buffs
2.  Switch to some other aura
3. Try and switch back to retribution aura, see spell cast but aura did not change

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by sebastienwood** [comment]
Confirmed on Kirin Tor

**Comment by TeridanDW** [comment]
Hello @cjwyatt , @sebastienwood .
There must be something else as simply entering a dungeon and cast retribution aura is not a problem at all.
![image](https://user-images.githubusercontent.com/6797611/33122175-278972fe-cf7f-11e7-9c47-11c09686c57c.png)

Closing untill furter information is provided.

**Comment by cjwyatt** [comment]
I've been taking more careful notice of when this happens and it seems like it works in most dungeons actually but since I've been looking out for it I've noticed it always breaks in Violet Hold.
